numpy/core/tests/test_multiarray.py

```diff
@@ -2850,6 +2850,9 @@ def test_temporary_with_cast(self):
         d = f.astype(np.float64)
         assert_equal(((f + f) + d).dtype, np.dtype('f8'))
 
+        c = np.ones(100000, dtype=np.complex)
```

Comment:
might make sense to move the elide test into its own class, its currently in binop but abs is a unary